recipe/bld.bat

```diff
@@ -21,22 +18,27 @@ cd build
 if errorlevel 1 exit /b 1
 
 :: Must add --use-local-env to NVCC_FLAGS otherwise NVCC autoconfigs the host
-:: compiler to cl.exe instead of the full path
-cmake %CMAKE_ARGS% .. ^
+:: compiler to cl.exe instead of the full path. MSVC does not support full
+:: C++11 standard
```

Comment:
As is, that comment is not correct. MSVC supports full C++17 since forever (VS2017 series), and C++11 since waaaaaaay back. The only problems that I can imagine you're running into is the interaction between msvc and nvcc - the latter is often further behind, but even there C++11 should be finde for CUDA 11+

Reply:
My mistake. I think I was confusing not implementing the standard with not allowing explicitly setting that standard because MSVC doesn't accept `/std:c++11`.

https://learn.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-160
https://learn.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=msvc-160